conf/tachyon-glusterfs-env.sh.template

```diff
@@ -54,7 +54,7 @@ export TACHYON_JAVA_OPTS+="
   -Dtachyon.data.folder=$TACHYON_UNDERFS_ADDRESS/tmp/tachyon/data
   -Dtachyon.workers.folder=$TACHYON_UNDERFS_ADDRESS/tmp/tachyon/workers
   -Dtachyon.worker.memory.size=$TACHYON_WORKER_MEMORY_SIZE
-  -Dtachyon.worker.data.folder=$TACHYON_RAM_FOLDER/tachyonworker/
+  -Dtachyon.worker.data.folder=/tachyonworker/
```

Comment:
it is relative path in each storage directory, not absolute path of the data path.